src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -1201,8 +1205,10 @@ void core::start_oxenmq() {
                 false,
                 m_pulse_thread_id);
         m_omq->add_timer([this]() { this->check_service_node_time(); }, 5s, false);
+        m_omq->add_timer([this]() { this->check_service_node_ip_address(); }, 15min, false);
```

Comment:
The use of `this->` here I find a bit strange (I do realize this is just copying the previous line):

```suggestion
        m_omq->add_timer([this]() { check_service_node_time(); }, 5s, false);
        m_omq->add_timer([this]() { check_service_node_ip_address(); }, 15min, false);
```